### PR TITLE
Fixing scroll position in notification

### PR DESCRIPTION
**Pull request by Rafi993**
## What type of PR is this?

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Fixes scroll position when loading more notifications

## Related Tickets & Documents

Closes https://github.com/forem/forem/issues/10714

## QA Instructions, Screenshots, Recordings

scrollbar

## Added tests?

- [ ] yes
- [x] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.forem.com
- [ ] readme
- [x] no documentation needed

### CODE REVIEW

**Comment by Rafi993** [comment]
Unrelated test is failing

File: app/assets/javascripts/initializers/initNotifications.js [hunk 1]
```diff
@@ -168,13 +168,19 @@ function initPagination() {
...
                 const newNotifications = document.createElement('div');
                 newNotifications.innerHTML = markup;
 
                 paginator.remove();
                 container.append(newNotifications);
 
                 initReactions();
+
+                // Restoring scroll position
+                document.documentElement.scrollTop = scrollTop;
```
Comment (Rafi993) [inline app/assets/javascripts/initializers/initNotifications.js:183]: I'm actually saving the scroll position and restoring it after loading more elements. Is this ok ?

**Comment by nickytonline** [comment]
@Rafi993, I'm not noticing this issue in production. Perhaps more recent design work fixed the issue?

Dec-08-2020 21-33-40

**Comment by Rafi993** [comment]
You are right @nickytonline I think this is resolved after new notification design was implemented. Please feel free to close this.

**Comment by nickytonline** [comment]
Closing this as the design rework of the notifications are appears to have fixed this.

**Pull request closed without merging**